bar-trender-front-end/src/components/Modals/ModalSearch.js

```diff
@@ -0,0 +1,255 @@
+
+import React, { Component } from "react";
+
+import { Col, Button, Form, FormGroup, Label, Input } from "reactstrap";
+import { Link } from "react-router-dom"
+import * as uuid from 'uuid';
+// reactstrap components
+import {
+  Modal,
+  ModalBody,
+
+} from "reactstrap";
+
+// core components
+import POSTForm from "../ApiFormSubmit";
+
+class ModalSearch extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      modal: props.initialModalState,
+      fade: true,
+
+      Paulaner: "off",
+      Triana: "off",
+      Alameda: "off",
+      Cruzcampo: "off",
+    };
+
+
+    this.handleTermChange = this.handleTermChange.bind(this);
+    this.handleSearch = this.handleSearch.bind(this);
+    this.handleEnter = this.handleEnter.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+
+    this.toggle = this.toggle.bind(this);
+  }
+
+  toggle() {
+    this.setState({
+      modal: !this.state.modal,
+      fade: !this.state.fade,
+      Paulaner: "off",
+      Triana: "off",
+      Alameda: "off",
+      Cruzcampo: "off",
+
+    });
+  }
+  handleTermChange(e) {
+    var checked_map = e.target.checked ? "on" : "off";
+    this.setState({ [e.target.name]: checked_map }, 
+      () => console.log( this.state, 'this.state'),
+    );
+    console.log( [e.target.checked], 'toggle value on');
+  }
+
+  async handleSearch(e) {
+    // this.props.searchEngine(this.state.term);
+
+
+    // console.log(this.state.filters);
+    // let temp_state = JSON.stringify(this.state.filters)
+
+    // console.log(this.state)
+
+    let beers = [];
+    let zones = [];
+
+    if (this.state['Paulaner'] == "on") {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7163535)
```suggestion
    if (this.state["Paulaner"] == "on") {
```